user/languages/csharp.md

```diff
@@ -121,6 +121,28 @@ script:
   - mono ./testrunner/xunit.runners.1.9.2/tools/xunit.console.exe ./MyPoject.Tests/bin/Release/MyProject.Tests.dll
 {% endhighlight %}
 
+#### Using solution-level nuget package
+Another way is to add the console testrunner of your choice as a solution-level nuget package.
+
+For many .net projects this will be the file found at:
+
+  ./.nuget/packages.config
```

Comment:
wrap it in backticks and integrate it with the preceding sentence, I think we don't really need it on a separate line.